rdkit/Chem/SaltRemover.py

```diff
@@ -171,16 +228,18 @@ def _applyPattern(m, salt, notEverything):
       return mol
```

Comment:
That is correct. It's essentially a convenience function to call `http://www.rdkit.org/Python_Docs/rdkit.Chem.rdmolops-module.html#DeleteSubstructs`
with predefined structures.